src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue

```diff
@@ -26,10 +26,29 @@
               :height="15" 
               color="base"
               class="d-inline-flex mr-1"
-            />starred
+              />starred
           </strong>
           areas.
         </p>
+        <div v-if="displayWarning" class="mb-4">
+          <ATATAlert
+              id="DOWLandingPageWarningAlert"
+              type="warning"
+              :showIcon="true">
+            <template v-slot:content>
+              <p class="mr-5 mb-0">
+                JWCC is not a support services contract, so you must define at
+                least one objective-based or cloud-specific requirement. Revisit
+                any of the starred performance areas to update your selection.
+                Once a requirement is added, all starred areas will be complete.
+                If you need assistance,
+                <a href="https://community.hacc.mil/s/contact" target="_blank">
+                  contact Customer Support
+                </a>.
```

Comment:
Please put period inside `<a>` tag to prevent period from wrapping by itself to new line at certain browser widths.

Reply:
Done